### DISCUSSION TITLE
I would like to ask, MikroORM under Mongo is the default start transaction?

### DISCUSSION THREAD

**Discussion by Lydanne** [mikro-orm/mikro-orm#2304] [Q&A] [answered]
**Describe the Issue**
I would like to ask, MikroORM under Mongo is the default start transaction?
Is the core of the transaction using the methods provided by Mongodb, or are you designing your own transaction system?

**Versions**

| Dependency | Version |
| - | - |
| node | ? |
| typescript | ? |
| mikro-orm | 4.5.9 |
| your-driver | ? |

**Comment by Lydanne** [comment]
You can still play like this. I learned.

**Comment by B4nan** [comment, marked as answer]
Yes, it uses the methods provided by `mongodb` client. You can see those in `MongoConnection` class.

https://github.com/mikro-orm/mikro-orm/blob/a74171119433a508cc6757a4e1f2e8bd5478b125/packages/mongodb/src/MongoConnection.ts#L194

**Reply by Lydanne** [reply]
Can I not use transaction? There was a problem when I used Mongoose transaction before, and the data was lost inexplicably.

**Reply by B4nan** [reply]
(implicit) transactions are disabled by default in mongo. Have you seen this page?

https://mikro-orm.io/docs/usage-with-mongo/#transactions

**Reply by Lydanne** [reply]
Thank you, I see.
